lib/api/core/plugins/pluginContext.js

```diff
@@ -295,12 +295,12 @@ function instantiateRequest(request, data, options = {}) {
     }
 
     for (const arg of Object.keys(_request.input.args)) {
-      if (!target.input.args[arg]) {
+      if (target.input.args[arg] === undefined) {
```

Comment:
I think you meant `!== undefined`

But then, undefined values cannot be iterated over using `for..of`, so this test can be removed entirely.